cmake/Kokkos_Version_Info.cpp.in

```diff
@@ -19,11 +19,12 @@
 namespace Kokkos {
 namespace Impl {
 
-std::string GIT_BRANCH             = "@GIT_BRANCH@";
-std::string GIT_COMMIT_HASH        = "@GIT_COMMIT_HASH@";
-std::string GIT_CLEAN_STATUS       = "@GIT_CLEAN_STATUS@";
-std::string GIT_COMMIT_DESCRIPTION = "@GIT_COMMIT_DESCRIPTION@";
-std::string GIT_COMMIT_DATE        = "@GIT_COMMIT_DATE@";
+std::string GIT_BRANCH       = "@GIT_BRANCH@";
```

Comment:
The branch name is also an arbitrary string. It should be made a raw string too.

Reply:
What character do you reckon is legal and would need to be escaped?

Reply:
Well apparently you are right, I managed to create a branch with a double quote character.  Will update the PR accordingly.